config/config.local.ts

```diff
@@ -0,0 +1,25 @@
+import * as Config from "webpack-chain";
+
+import * as CommonConfig from "./config.common";
+import { EnvOptions } from "./types";
+
+function webpackConfig(options: EnvOptions = {}): Config {
+  // get the common configuration to start with
+  const config = CommonConfig.init(options);
+
+  const localPath = "/home/USER_NAME/.config/Screeps/scripts/127_0_0_1___21025/default/";
```

Comment:
I think this could be moved to our `credentials.json` or some other gitignored file. Just in case we have people running private instances of Screeps on a private server separate from their local machine.